content/blog/introducing-python-sdk-for-dscc.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Introducing Python SDK for Data Services Cloud Console
+date: 2023-03-29T13:45:50.979Z
+priority: 0
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+In my previous [blog](https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/), I discussed how to generate a Software Development Kit (SDK) from the Data Services Cloud Console Open API spec using a third-party tool called OpenAPI generator. Today, I am going to talk more in detail on the Python SDK generated using the tool. As we progress, we will discuss the implementation aspects, and then take up a use-case to better understand how to use the SDK.
+
+**APIs and SDKs** 
+
+APIs are a list of functions libraries that are used to communicate with a web service (via secure HTTP), whereas an SDK is a development kit that facilitates the usage of these APIs. APIs enable any software developer to create business opportunities by leveraging the capabilities provided by the API to extend their application whereas the SDK facilitates the process for developers. An SDK is a collection of software tools and programs for a specific application platform that allows developers to manipulate the functions supported by the service. These can be considered as a wrapper on the top of the APIs, making the code consumable by the application.
+
+One thing to note is that, the user always gets access to the latest Data Services Cloud Console API version through the SDKs. How? The SDK is designed and deployed (using CI/CD pipelines such as Jenkins) in such a way that with every new release of the Data Services Cloud Console Open API spec, the SDKs get updated automatically. Thus, keeping it up-to date without any manual intervention. This also reduces time which is spent waiting for updates with newer features.
+
+**Introducing Python SDK for Data Services Cloud Console**
+
+Due to the wide adoption of Python, and for the Python lovers out there who did not have an option to achieve their automation goals, we have the Python SDK available now. You can access the SDK onthis [github ](https://github.com/HewlettPackard/greenlake-data-services-python)page.
+
+This SDK contains the following:
+
+* Documentation (under docs folder)
+* Code libraries (under greenlake_data_services folder)
```

Comment:
Check the formatting of "greenlake_data_services" folder in the preview. Right now, it looks like this:
![image](https://user-images.githubusercontent.com/77496261/228363830-86bdb772-bd53-4b4e-af05-42838b92423a.png)
Not sure that's what you wanted.

Reply:
There is an issue with underscore and would be resolved by Denis before publishing.